training/SYMBOL_TABLE_OVERVIEW.md

```diff
@@ -0,0 +1,170 @@
+# Overview
+
+The purpose of this document is to serve as a high-level guide to the structure and elements within Arkouda's Symbol Table. This guide does not cover creating or refactoring new Symbol Table Entry types. 
+
+For more information regarding the creation of new Symbol Table Entry types, view the [Symbol Table Entry Creation](https://github.com/Bears-R-Us/arkouda/blob/master/training/SYMBOL_TABLE_ENTRY_CREATION.md) document.
+
+<a id="toc"></a>
+## Table of Contents
+1. [Symbol Table Overview](#overview)
+2. [Symbol Table Structure](#symStruct)
+3. [Creating a New Entry From an Array](#addSym)
+4. [Finding an Entry in the Symbol Table](#findSym)
+5. [Accessing Symbol Table Entry Data](#accessSym)
+
+<a id="overview"></a>
+## Symbol Table Overview
+
+The Symbol Table is implemented using a Chapel Map and serves as a `lookup table` that uses a Chapel `Map` which allows Arkouda to persist objects until the server is shutdown or the object is deleted.

```

Comment:
```suggestion
The Symbol Table is implemented using a Chapel Map and serves as a `lookup table` which allows Arkouda to persist objects until the server is shutdown or the object is deleted.

```

Reply:
I think @pierce314159 suggestion makes sense here to avoid repeating that a Chapel Map is used.

Reply:
Updated to use this suggestion